detect/synthetic-monitoring/api-checks/configuration.mdx

```diff
@@ -1,32 +1,26 @@
 ---
-title: 'Requests'
-description: 'Requests'
-sidebarTitle: 'Requests'
+title: 'Configuration'
+description: 'Configure your API checks to validate responses and ensure your endpoints behave as expected.'
+sidebarTitle: 'Configuration'
+aliases:
+  - /detect/synthetic-monitoring/api-checks/assertions/
+  - /detect/synthetic-monitoring/api-checks/requests/
```

Comment:
Do these make sure that the old links still work? For example, the CLI repo links to various places in the documentation and we should try to avoid breaking them.